templates/plugin/remote_execution_ssh.yml.erb

```
@@ -16,4 +16,14 @@
 
 :mqtt_broker: <%= scope.lookupvar("::foreman_proxy::plugin::remote_execution::mosquitto::broker") %>
 :mqtt_port: <%= scope.lookupvar("::foreman_proxy::plugin::remote_execution::mosquitto::port") %>
+
+<% mqtt_ttl = scope.lookupvar('::foreman_proxy::plugin::remote_execution::script::mqtt_ttl'); if mqtt_ttl -%>
```

A more common way to write this:
```suggestion
<% if (mqtt_ttl = scope.lookupvar('::foreman_proxy::plugin::remote_execution::script::mqtt_ttl')) -%>
```